src/modules/media/utils.ts

```diff
@@ -22,7 +22,7 @@ export function dataURLToBlob(dataUrl: string): Blob | null {
 }
 
 export async function objectURLToBlob(objectURL: string): Promise<Blob> {
-  return fetch(objectURL).then(res => res.blob())
+  return fetch(objectURL, { headers: { pragma: 'no-cache', 'cache-control': 'no-cache' } }).then(res => res.blob())
```

Comment:
Maybe abstract the headers into a separate object/function to avoid repetition

Reply:
```ts
const noCacheHeaders = { pragma: 'no-cache', 'cache-control': 'no-cache' }
```

Reply:
or even wrap the fetch function
```ts
const noCacheFetch = (url: Parameters<typeof fetch>[0], init: Parameters<typeof fetch>[1] = {}) =>
  fetch(url, { ...init, headers: { ...init?.headers, pragma: 'no-cache', 'cache-control': 'no-cache' } })
```
But this might be overkill